bindings/python/repair_wheel.py

```diff
@@ -0,0 +1,83 @@
+import atexit
+import glob
+import os
+import sys
+import tempfile
+from subprocess import run
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+wheel_dir, wheel_file, delocate_args = sys.argv[1:]
+wheel_dir = wheel_dir.replace(os.sep, "/")
+wheel_file = wheel_file.replace(os.sep, "/")
+
+# Ensure pyarrow.
+if "universal2" in wheel_file:
```

Comment:
Why can't we rely on regular pip install like we do for other platforms? Could you add a comment?

Reply:
Done